pkg/cvo/status.go

```diff
@@ -327,6 +327,14 @@ func updateClusterVersionStatus(cvStatus *configv1.ClusterVersionStatus, status
 		failingCondition.Reason = failingReason
 		failingCondition.Message = failingMessage
 	}
+	if failure != nil &&
+		skipFailure &&
+		progressReason == "ClusterOperatorUpdating" &&
```

Comment:
I think this can actually be `ClusterOperatorsUpdating` (notice the plural) when there are more operators updating, which would prevent us from entering this branch - we'd need to handle this somehow.